doc/developing-packages.rst

```diff
@@ -259,6 +259,54 @@ your libraries and executable sections. For example:
 Note that the ``import`` **must** be the first thing in the stanza. For more
 information see the :ref:`common-stanzas` section.
 
+Sets of packages that are verified to build together
+----------------------------------------------------
+
+Since Cabal 3.8, you can import from a
+:ref:`remote project configuration file<conditionals and imports>`:.
+This frees you from having to determine the acceptable version bounds for your
+dependencies, as described in the previous section.
+
+One provider of package sets is Stackage_. The Stackage snapshots contain a set
+of packages from Hackage that have all been verified to build with a given
+version of GHC.
+
+For example, the snapshot lts-19.2 contains versioned packages which all compile on GHC 9.0.
+
+If you use the following ``cabal.project`` file, the packages of the snapshot
+would be constrained to specific versions. The Cabal solver is then liberated
+from determining which versions to use, since it has already been determined
```

Comment:
Not sure if `liberated` would be appropiate here. In fact the solver is `constrained` to use those versions, no?

Reply:
Ok, I have removed this part in the new version of the PR which is af9cb087850b759f32ae0fad565cf4924650f3d7